Noura/c5.js

```diff
@@ -0,0 +1,10 @@
+//Write down a function that gets an array of number from the user and then sort the number from the lowest to the highest.
+
+let numbers = [0, 1 , 2, 54, 10, 20, 3 ];
+numbers.sort( function(a , b){
```

Comment:
Please use another approach for this question. Do not use the array's `sort` method.